packages/babel-cli/src/babel/file.js

```diff
@@ -91,25 +104,27 @@ export default async function({ cliOptions, babelOptions }) {
     }
   }
 
-  function readStdin() {
-    return new Promise((resolve, reject) => {
-      let code = "";
+  function readStdin(): Promise<string> {
+    return new Promise(
+      (resolve: Function, reject: Function): void => {
+        let code = "";
 
-      process.stdin.setEncoding("utf8");
+        process.stdin.setEncoding("utf8");
 
-      process.stdin.on("readable", function() {
-        const chunk = process.stdin.read();
-        if (chunk !== null) code += chunk;
-      });
+        process.stdin.on("readable", function() {
+          const chunk = process.stdin.read();
+          if (isString(chunk)) code += toString(chunk);
```

Comment:
We don't need `isString` and `toString` here.

Note that `chunk` could also be a `Buffer` ([docs](https://nodejs.org/dist/latest-v12.x/docs/api/stream.html#stream_event_data)), so `isString(chunk)` is not equivalent to `chunk !== null`.

`code += chunk` will implicitly convert `chunk` to string since `code` is a string.

Reply:
I can remove `isString`, but without `toString` flow will throw an error because it sees `Buffer` being incompatible with `String`

Reply:
We can `// $FlowIgnore` it

Reply:
Alright.